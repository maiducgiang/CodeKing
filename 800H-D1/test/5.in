10
19
11 4 1 7 8 15 12 2 12 17 10 13 17 14 4 11 18 8 2 
10
3 2 6 10 7 9 7 7 3 3 
12
9 5 2 5 12 9 3 12 5 2 12 6 
22
22 20 9 5 6 11 18 5 7 1 21 8 18 18 19 12 15 13 13 5 6 5 
19
4 4 3 14 9 2 5 9 7 17 12 11 15 12 3 15 8 18 15 
11
5 2 5 7 5 8 6 5 9 5 2 
27
20 26 8 6 11 6 1 26 2 10 26 19 17 7 17 27 27 1 14 12 26 11 2 9 17 25 2 
29
26 26 16 7 27 17 27 5 21 16 28 2 22 19 11 2 29 21 7 23 6 1 9 11 10 10 22 8 22 
24
8 3 13 15 8 24 9 19 10 15 20 10 15 14 10 24 9 2 1 23 5 14 19 17 
15
9 9 9 15 11 14 1 11 11 14 2 11 8 15 14 
