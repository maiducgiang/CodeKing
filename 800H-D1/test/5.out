3
4
5
10
10
3
11
13
9
9
